docs/src/install-guide.md

```diff
@@ -1,146 +1,18 @@
 # Installation
 
-To install the latest version of Kani, run:
+There are two ways to install Kani:
 
-```bash
-cargo install --locked kani-verifier
-cargo-kani setup
-```
+1. By downloading
+[pre-compiled release binaries](./install-pre-compiled.md).
+This is suitable for Kani users who don't
+expect to change the source code.
+2. By [compiling the source code](./install-from-source.md).
+This is good for Kani developers.
```

Comment:
I would say that the option for building from source is more suitable for Kani developers or users who want to try the latest changes (as you say in the "Build from source" section). You can have it as an extra sentence in the bullet point or later. The other option should sound like the default one.

Reply:
Fixed the wording here.